arbitrum-docs/node-running/running-a-node.md

```diff
@@ -103,4 +103,7 @@ Note: If you’re interested in accessing an Arbitrum chain, but you don’t wan
 - `--node.rpc.gas-cap`
   - Defaults to `50000000`, cap on computation gas that can be used in `eth_call`/`estimateGas` (0 = no cap)
 - `--node.rpc.tx-fee-cap`
-  - Defaults to `1`, cap on transaction fee (in ether) that can be sent via the RPC APIs (0 = no cap)
\ No newline at end of file
+  - Defaults to `1`, cap on transaction fee (in ether) that can be sent via the RPC APIs (0 = no cap)
+- `--ipcpath`
+  - Filename for IPC socket/pipe within the datadir (explicit paths escape it)
+  🔉 Note that `IPC` is not supported on the macOS yet, and hence, it will return connection refused if used.
```

Comment:
Also we can add a note that this path is within the docker container, users should put it to the docker mount point.